Package.swift

```diff
@@ -5,24 +5,34 @@ let package = Package(
     name: "MiniApp",
     defaultLocalization: "en",
     platforms: [
-        .iOS(.v13)
+        .iOS(.v14)
     ],
     products: [
-        .library(
-            name: "MiniApp",
-            targets: ["MiniApp"])
+        .library(name: "MiniApp", targets: ["MiniApp"])
     ],
     dependencies: [
         .package(url: "https://github.com/weichsel/ZIPFoundation.git", .upToNextMajor(from: "0.9.0")),
-        .package(url: "https://github.com/datatheorem/TrustKit", .upToNextMajor(from: "2.0.0")),
-        .package(url: "https://github.com/rakutentech/GoogleMobileAds-SPM.git", .upToNextMajor(from: "8.13.0"))
+        .package(url: "https://github.com/datatheorem/TrustKit.git", .upToNextMajor(from: "2.0.0")),
+        .package(name: "GoogleMobileAds-SPM", url: "https://github.com/rakutentech/GoogleMobileAds-SPM.git", .upToNextMajor(from: "8.0.0")),
+        .package(url: "https://github.com/rakutentech/ios-analytics-framework.git", .upToNextMajor(from: "8.0.0"))
```

Comment:
@laubengaier I added dependencies for RAT, you forgot them ^^
@rleojoseph @corycaywood @laubengaier Be careful there, I locked Google ads to v8 because Link 2.11 is not up to date with version 9. I also locked RAT to v8 for the same reason, but a v9 is already in place